src/common/Backend.ts

```diff
@@ -267,6 +267,7 @@ export interface BackendEventMap {
     'backend-status': {
         message: string;
         percent: number;
+        statusPercent?: number;
```

Comment:
I just noticed this. Neither of those numbers are percentages (0-100). They are both 0-1. Maybe we should just rename them to `totalProgress` and `statusProgress` to be consistent with my `Progress` interface (TS).

Also, the type of `statusPercent` is wrong. It can be null, so it has to be `statusPercent?: number | null`.